src/interfaces/special-resolution-interfaces/create-resolution-interface.ts

```diff
@@ -0,0 +1,10 @@
+import { PersonIF } from '@/interfaces'
+
+export interface CreateResolutionIF {
+  // validationDetail: ValidationDetailIF
+  resolutionConfirmed: boolean
+  resolutionDate?: string
+  resolutionText?: string
+  signingPerson?: PersonIF
+  signingDate?: string
```

Comment:
It would be helpful to document, beside each date (lines 6 and 9), what format they are, eg, `// YYYY-MM-DD`.